File: php/tutorial/05-usespeechvstone.php
<?php
// -----------
// using speech input instead of touch-tone
// -----------


answer();

$result = ask( "For sales, just say sales.  For support, say support.", 
			array("choices" => "sales, support", "repeat" => 3) );

if ($result->name=='choice')
{
	if ($result->value=="sales")   say( "sales is not available right now."       );
	if ($result->value=="support") say( "support is currently on the other line." );
}

hangup();
?>

File: ruby/tutorial/core/05-usespeechvstone.rb
# -----------
# using speech input instead of touch-tone (DTMF)
# -----------

answer

options = { :choices => 'sales, support', 
            :repeat  => 3 }
            
result = ask 'For sales, just say sales.  For support, say support.', options

if result.name == 'choice'
  case result.value
  when 'sales'
    say 'sales is not available right now.'
  when 'support'
    say 'support is currently on the other line.'
  end
end

hangup